modules/end-to-end-test/src/test/groovy/org/hawkular/inventory/rest/test/InventoryITest.groovy

```diff
@@ -157,14 +185,14 @@ class InventoryITest extends AbstractTestBase {
         Collections.reverse(entries)
         for (Map.Entry en : entries) {
             String path = en.getKey();
-            boolean validate = en.getValue();
+            String getValidationPath = en.getValue();
```

Comment:
Yes, the name may resemble a getter method. Clearly, the meaning of the name is "the path to use when checking that the deletion was successful" - do you have any better idea?